src/Database/DBView.php

```diff
@@ -0,0 +1,142 @@
+<?php
+/**
+ * @copyright Copyright (C) 2020, Friendica
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Friendica\Database;
+
+use Exception;
+use Friendica\Core\Hook;
+use Friendica\Core\Logger;
+use Friendica\DI;
+use Friendica\Util\DateTimeFormat;
+use phpDocumentor\Reflection\Types\Boolean;
+
+require_once __DIR__ . '/../../include/dba.php';
+
+class DBView
```

Comment:
This class is already in the `Database` namespace, so it doesn't need to be called `DB`, it can just be called `View`.

Reply:
I was thinking about this as well. But the related class is called `DBStructure`, the associated file to that is called `dbstructure.config.php`. So I thought it would be inconsistent to call it `View` and the config file `dbview.config.php`, but naming that file just `view.config.php` hadn't seemed alright to me as well.

Reply:
The name `dbstructure` is a leftover from before namespaces and as such is justified to include `DBA`, but `Friendica\Database\View` is brand-new.

Reply:
How should the config file then be named? And shouldn't we rename the `DBStructure` then as well?

Reply:
`static/dbview.config.php` is fine because of the lack of namespace, but yeah `DBStructure` could just be `Friendica\Database\Structure`.

Reply:
I had a quick look at it. I prefer not to do this rename in this PR :-)

Reply:
It's fine, I wasn't asking to rename `DBStructure` in the first place.